desktop/src/main/java/bisq/desktop/main/overlays/windows/TacWindow.java

```diff
@@ -72,8 +72,7 @@ public void show() {
 
                 "2. The user is responsible for using the software in compliance with local laws. Don't use the software if using it is not legal in your jurisdiction.\n\n" +
 
-                "3. The " + Res.getBaseCurrencyName() + " market price is delivered by 3rd parties (BitcoinAverage, Poloniex, Coinmarketcap). " +
-                "It is your responsibility to verify the price with other sources for correctness.\n\n" +
+                "3. Any " + Res.getBaseCurrencyName() + " market prices, network fee estimates, or other data obtained from servers operated by the Bisq DAO is provided is on an 'as is, as available' basis without representation or warranty of any kind. It is your responsibility to verify any data provided in regards to inaccuracies or omissions.\n\n" +
```

Comment:
> .. is provided is on an ...
=> There's an extra "is"

Reply:
Thanks and good catch, but for some reason I'm unable to approve your suggested change in the GitHub app - will fix it later